include/dsAudio.h

```diff
@@ -1742,9 +1507,9 @@ dsError_t dsAudioAtmosCapsChangeRegisterCB (dsAtmosCapsChangeCB_t cbFun);
 dsError_t dsGetAudioCapabilities(intptr_t handle, int *capabilities);
 
 /**
- * @brief Gets the MS12 capabilities of audio port supported by the platform.
+ * @brief Gets the MS12 capabilities supported by the platform.
  * 
- * This function is used to get the supported MS12 capabilities for the input port supported by the platform.
+ * This function is used to get the supported MS12 capabilities of the platform.
```

Comment:
Do we need to mention that this API is port independent as mentioned in other APIs?

Reply:
updated